### DISCUSSION TITLE
Update Return URL and/or Enable AJAX tooltips

### DISCUSSION THREAD

**Discussion by jrrdnx** [solspace/craft-freeform#180] [Feature Requests]
We've had some clients who have had issues with the Return URL field not working correctly, and it turns out it's because they have the Enable AJAX option checked. Could the tooltips for one or both of these fields be updated to note that the Return URL field is ignored if AJAX is enabled? Thanks!

**Comment by kjmartens** [comment]
Thanks! I've added a note to both of these in the builder tooltips now.

Also, in case you weren't already aware, there is a way to allow the Return URL to still work when AJAX is enabled. You can see that here:
https://docs.solspace.com/craft/freeform/v3/guides/success-return-on-forms.html#forcing-a-different-success-url-on-ajax-forms

Essentially, adding something like:

``` html
<script>
  // Find the form element
  const form = document.getElementById('my-form');
  // Do something on a successful ajax submit
  form.addEventListener('freeform-ajax-success', function(event) {
    // Redirect the user
    window.location.href = event.response.returnUrl;
})
</script>
```

**Reply by jrrdnx** [reply]
Thanks @kjmartens!

I did see that in the documentation and we're looking at making that work dynamically with all forms!
